code/modules/ventcrawl/ventcrawl.dm

```diff
@@ -0,0 +1,210 @@
+var/list/ventcrawl_machinery = list(
+	/obj/machinery/atmospherics/unary/vent_pump,
+	/obj/machinery/atmospherics/unary/vent_scrubber
+	)
+
+// Vent crawling whitelisted items, whoo
+/mob/living/var/list/can_enter_vent_with = list(
```

Comment:
I wonder if it'd be easier to use an item's `w_class` instead, and to permit only values of 1 and maybe 2?